include/swift/AST/Decl.h

```diff
@@ -4517,6 +4521,8 @@ class AbstractFunctionDecl : public ValueDecl, public DeclContext {
 
   ImportAsMemberStatus IAMStatus;
 
+  BodyPostProcessor PostProcessor = nullptr;
```

Comment:
Also totally fine going into a subsequent PR, but could we use a bit (up in `AbstractFunctionDeclBitfields`) to indicate the presence of a `BodyPostProcessor` and then put the actual post-processors into a `DenseMap` on `ASTContext`? It's a shame to waste a word in every function for these rare cases.